front/text/plain/convert.go

```diff
@@ -132,11 +132,12 @@ func ToHTML(text string) string {
 		return text
 	}
 
```

Comment:
The strings.Split() call can be replaced with something more efficient if `lines` is unused